test/cli.test.js

```diff
@@ -58,45 +58,3 @@ it('should correctly print help when unknown action is used', async () => {
   expect(stdout).toMatch('$ nft [command] <file>');
 });
 
-it('[codecov] should correctly print trace from required cli', async () => {
-  // This test is only here to satisfy code coverage
-  const cli = require('../out/cli.js')
-  const files = join(__dirname, inputjs);
-  const stdout = await cli('print', files);
-  expect(stdout).toMatch(normalizeOutput(outputjs));
-});
-
-it('[codecov] should correctly build dist from required cli', async () => {
-  // This test is only here to satisfy code coverage
-  const cli = require('../out/cli.js')
-  const files = join(__dirname, inputjs);
-  await cli('build', files);
-  const found = existsSync(join(__dirname, outputjs));
-  expect(found).toBe(true);
-});
-
-it('[codecov] should correctly show size in bytes from required cli', async () => {
-  // This test is only here to satisfy code coverage
-  const cli = require('../out/cli.js')
-  const files = join(__dirname, inputjs);
-  const stdout = await cli('size', files);
-  expect(stdout).toMatch('bytes total');
-});
-
-it('[codecov] should correctly show why from required cli', async () => {
-  // This test is only here to satisfy code coverage
-  const cli = require('../out/cli.js')
-  const entrypoint = join(__dirname, inputjs);
-  const exitpoint = join(__dirname, outputjs);
-  const cwd = __dirname;
-  const stdout = await cli('why', entrypoint, exitpoint, 'dist', cwd);
-  expect(stdout.replace(/\\/g, '/')).toMatch('unit/wildcard/assets/asset1.txt\nunit/wildcard/input.js');
-});
-
-it('[codecov] should correctly print help when unknown action is used', async () => {
-  // This test is only here to satisfy code coverage
-  const cli = require('../out/cli.js')
-  const files = join(__dirname, inputjs);
-  const stdout = await cli('unknown', files);
-  expect(stdout).toMatch('$ nft [command] <file>');
-});
```

Comment:
I think we should keep these tests because we can still calculate code coverage without sending it to the codecov service (thats why CI is failing)

Reply:
<picture data-single-emoji=":ack:" title=":ack:"><img class="emoji" width="20" height="auto" src="https://single-emoji.vercel.app/api/emoji/eyJhbGciOiJkaXIiLCJlbmMiOiJBMjU2R0NNIn0..YFqVtC7Hcbwwg9_e.JBJXTPn6Z3wUr2BTuR1O-fNFJNfOw43nLZKSGkKel56EkzwOMagClMaJ-vEbPj4nSA-UTMmTH49FyZFXLFi323VcU9JGgQXUI0QGOOwiNIJLypsvlQ.fnPCFrMYkVfqXP_teRGOcA" alt=":ack:" align="absmiddle"></picture> <picture data-single-emoji=":done:" title=":done:"><img class="emoji" width="20" height="auto" src="https://single-emoji.vercel.app/api/emoji/eyJhbGciOiJkaXIiLCJlbmMiOiJBMjU2R0NNIn0..wTy9yOCS_c4Hglc7.9MT2-NQtK0hIBMtMRPhNC0og_P6EL6XFoyt4CRq3Bu5aFmJTMst30rWFglBzlVE3FBqPyjFx6SpyW3_d3DrzYh91kuQdykpKPR369J5yp-iyIMSgtB0.X6QoxiDtWlxbypzGXXiQoQ" alt=":done:" align="absmiddle"></picture>